salt/modules/aixpkg.py

```diff
@@ -0,0 +1,403 @@
+# -*- coding: utf-8 -*-
+'''
+Package support for AIX
+
+.. important::
+    If you feel that Salt should be using this module to manage filesets or
+    rpm packages on a minion, and it is using a different module (or gives an
+    error similar to *'pkg.install' is not available*), see :ref:`here
+    <module-provider-override>`.
+'''
+# Import python libs
+from __future__ import absolute_import, print_function, unicode_literals
+import os
+import copy
+import logging
+
+
+# Import salt libs
+import salt.utils.data
+import salt.utils.functools
+import salt.utils.path
+import salt.utils.pkg
+from salt.exceptions import CommandExecutionError
+
+
+log = logging.getLogger(__name__)
+
+# Define the module's virtual name
+__virtualname__ = 'pkg'
+
+
+def __virtual__():
+    '''
+    Set the virtual pkg module if the os is AIX
+    '''
+    if __grains__['os_family'] == 'AIX':
+        return __virtualname__
+    return (False,
+            'The aixpkg execution module failed to load.')
```

Comment:
I don't think this is a good error message. `failed` suggests that it should have loaded and did not. Perhaps something like `Did not load AIX module on non-AIX OS`?